changelog/unreleased/disable-versioning.md

```diff
@@ -0,0 +1,4 @@
+Enhancement: Add HTTP header to disable versioning on EOS
+
+This enhancement introduces a new header, `X-Disable-Versioning`, on PUT requests. Whenever this header is set, with any value (but `true` is recommended), EOS will not version this file save.
```

Comment:
Is it really any value? Isn't any value convertible by `ParseBool`? I would rephrase for something like

```suggestion
This enhancement introduces a new header, `X-Disable-Versioning`, on PUT requests. EOS will not version this file save whenever this header is set with a truthy value.
```